OcCo_TF/train_completion.py

```diff
@@ -75,8 +75,9 @@ def train(args):
 
     is_training_pl = tf.placeholder(tf.bool, shape=(), name='is_training')
     global_step = tf.Variable(0, trainable=False, name='global_step')
-    alpha = tf.train.piecewise_constant(global_step, [10000, 20000, 50000],
-                                        [0.01, 0.1, 0.5, 1.0], 'alpha_op')
+    alpha = 0
```

Comment:
Perhaps a comment would be nice to document why you're doing this. _I_ know why, but it would be good for a future person working on this repo.